index.html

```diff
@@ -0,0 +1,82 @@
+                <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+                <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</li>
+            </ul>
+        </section>
+
+        <section id="What_you_should_already_know" class="main-section">
+            <header>What you should already know</header>
+            <p>A general understanding of the Internet and the World Wide Web (WWW).</p>
+            <p>Good working knowledge of HyperText Markup Language (HTML).</p>
+            <code>x = 2;</code>
+            <ul>
+                <li>A general understanding of the Internet and the World Wide Web (WWW)</li>
+                <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+                <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+            </ul>
+        </section>
+
+        <section id="JavaScript_and_Java" class="main-section">
+            <header>JavaScript and Java</header>
+            <p>JavaScript and Java are similar in some ways but fundamentally different in some others.</p>
+            <p>The JavaScript language resembles Java but does not have Java's static typing and strong type checking.</p>
+            <code>x = 3;</code>
+        </section>
+        <section id="Hello_world" class="main-section">
+            <header>Hello world</header>
+            <p>To get started with writing JavaScript, open the Scratchpad and write your first "Hello world" JavaScript code</p>
+            <p>Select the code in the pad and hit Ctrl+R to watch it unfold in your browser!</p>
+            <code>x = 4;</code>
+        </section>
+        <section id="Variables" class="main-section">
+            <header>Variables</header>
+            <p>You use variables as symbolic names for values in your application. The names of variables, called identifiers, conform to certain rules.</p>
+            <p>A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also be digits (0-9).</p>
+            <code>x = 5;</code>
+        </section>
+    </main>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please kindly remove the words that does not define a portion in your code.

Reply:
Nice job using `github` inline comments feature on your comments👍. But let's make this comment even better by stating why you are requesting the change😉. remember you may be aware of the reason but not for someone else😉. don't foget we learnt this in our lesson on [how to give a code review😉.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)

<img width="663" alt="image" src="https://user-images.githubusercontent.com/64643995/235462893-7663ee87-fb88-4631-bec3-443b6960ef33.png">

Reply:
Please kindly remove the words that does not define a portion in your code as comments. Those words as comments makes your code look unprofessional.